docusaurus/docs/identities/enrolling.md

```diff
@@ -80,8 +80,9 @@ ziti edge enroll \
 ### 3rd Party CA - Auto
 
 When using a third party CA identity creation process in the Ziti Controller is
-automatic. The act of enrolling the identity will create it. Like "3rd Party CA

```

Comment:
> with this "auto" flow

extra words that don't help with clarity. remove those, please?

"The act of enrolling" --> "Enrolling the identity will create it..."